### DISCUSSION TITLE
How to handle indirect calls in LTO mode？

### DISCUSSION THREAD

**Discussion by iorra-cifer** [AFLplusplus/AFLplusplus#2078] [Q&A]
Hello, I am a beginner. I would like to know how AFL++'s LTO mode handles introduction calls?
I saw in the SanitizerCoverageLTO.so.cc file that 
`SanCovTracePCIndir = M.getOrInsertFunction(SanCovTracePCIndirName, VoidTy, IntptrTy);`
But I couldn't find the implementation of __ sanitizer_cov_trace_pc_indir.
I also want to know if there is a coverage loss issue with Sanitizer Coverage using LLVM, as it simulates edges by inserting basic blocks.

**Comment by vanhauser-thc** [comment]
We are not instrumenting calls except in ctx mode. In ctx we instrument calls but not indirect ones
